esmvalcore/config/_config_validators.py

```diff
@@ -4,10 +4,12 @@
 import logging
 import os.path
 import warnings
-from collections.abc import Iterable
+from collections.abc import Callable, Iterable
 from functools import lru_cache, partial
 from pathlib import Path
-from typing import Any, Optional, Union
+from typing import TYPE_CHECKING, Any, Optional, Union
+
+from packaging import version
```

Comment:
Don't forget to add the new dependency to setup.py/environment.yml

Reply:
Good find, I thought it's already there. I also noticed that it's missing in ESMValTool, issue: https://github.com/ESMValGroup/ESMValTool/issues/3059.